ui/style.css

```
@@ -117,3 +117,12 @@ td.more {
 td.more:hover {
     background-color: #eee;
 }
+
+div.event_status {
+    display: flex;
+    justify-content: space-between;
+}
+
+div.event_status span {
+    color: red;
```

The icon looks very thin and fragile. Maybe just make it a little bit larger?

```suggestion
    color: red;
    font-size: larger;
```